lld/COFF/Driver.cpp

```diff
@@ -368,111 +368,116 @@ bool LinkerDriver::isDecorated(StringRef sym) {
 // Parses .drectve section contents and returns a list of files
 // specified by /defaultlib.
 void LinkerDriver::parseDirectives(InputFile *file) {
```

Comment:
Can you add another function above this one, which iterates on the directives vector and calls this one, but with an extra `s` arg? And probably make this function private.

Reply:
As you've suggested, but with a name change to further distinguish section processing from argument parsing. Too many `parseDirectives` meaning different things, imho.